servers/v3-proxy-api/src/middleware/errorHandlers.ts

```diff
@@ -0,0 +1,104 @@
+import { serverLogger } from '@pocket-tools/ts-logger';
+import { NextFunction, Request, Response } from 'express';
+import * as Sentry from '@sentry/node';
+import { ClientError } from 'graphql-request';
+import { InputValidationError } from '../errors/InputValidationError';
+
+const errorHeaders = {
+  INTERNAL_SERVER_ERROR: {
+    'X-Error-Code': 198,
+    en: 'Internal Server Error',
+  },
+  FORBIDDEN: {
+    'X-Error-Code': 5200,
+    en: 'Forbidden',
+  },
+  BAD_USER_INPUT: {
+    'X-Error-Code': 130,
+    en: 'Bad request',
+  },
+  GRAPHQL_VALIDATION_FAILED: {
+    'X-Error-Code': 130,
+    en: 'Bad request',
+  },
+};
+
+/**
+ * (Partial) implementation of v3 API's custom error headers.
+ * See public_html/localization/dict-errors-*.json in Web repo.
+ */
+export function customErrorHeaders(errorCode: unknown, language = 'en') {
+  if (
+    typeof errorCode !== 'string' ||
+    errorCode == null ||
+    errorHeaders[errorCode] == null
+  ) {
+    return undefined;
+  }
+  //todo: proxy should handle localization based on web repo request.
```

Comment:
For the time being i think returning only english to devs for now is fine

Reply:
github tricks - this is an old comment, just unindented I think?